src/js/App/Header/LogoutAlert.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import cookie from 'js-cookie';
+
+class LogoutAlert extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            alertOneVisible: cookie.get('cs_loggedOut') === 'true',
+            delay: 5000
+        };
+    }
+
+    hideAlertOne = () => {
+        this.setState({ alertOneVisible: false });
+        cookie.set('cs_loggedOut', 'false');
+    }
+    componentDidMount() {
+        this.setTimer();
+    }
+
+    setTimer = () => {
```

Comment:
It's kinda interesting to hide the alert after 5s, but PF doesn't like auto closing alerts and since you are already using `AlertActionCloseButton` no need to close after some time as well. So we should remove this.

Reply:
PF has in their guidelines that autoclosing alerts are fine. They recommend 8 seconds instead of 5, so we can change it to that. But, the alert should close by itself according to the mocks.

Reply:
done